index.html

```diff
@@ -59,21 +59,22 @@ <h1 class="pink-highlight">Yargs be a node.js library fer hearties tryin' ter pa
 
         <pre>
           <code class="hljs language-javascript">
-  #!/usr/bin/env node
-
-  require('yargs')
-    .usage('$0 &lt;cmd&gt; [args]')
-    .command('hello [name]', 'welcome ter yargs!', (yargs) => {
-      yargs.positional('name', {
-        type: 'string',
-        default: 'Cambi',
-        describe: 'the name to say hello to'
-      })
-    }, function (argv) {
-      console.log('hello', argv.name, 'welcome to yargs!')
+#!/usr/bin/env node
+
+require('yargs')
+  .scriptName("pirateParser")
```

Comment:
let's make this `pirate-parser` rather than `pirateParser`, just thinking of the naming conventions I'd usually follow for bins.